### PR TITLE
Update truffle frontend

### PR DESCRIPTION
**Pull request by mdhm98**

### CODE REVIEW

File: backend/contracts/Habit.sol [hunk 1]
```diff
@@ -12,15 +12,15 @@ contract Habit {
...
-    address owner;
+    address owner = msg.sender;
     uint start_time;
     uint end_time;
     uint256 pool;
 
 
     // Create the challenge and set a date range
     // client side will convert date/time to block.timestamp offset
-    constructor(uint start_time_) public payable {
```
Comment (dragontho) [inline backend/contracts/Habit.sol:23]: I'll be changing this to reflect the Dice a bit more, without constructor and having a mapping of habits instead.

File: backend/migrations/2_deploy_contracts.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+var Habit = artifacts.require("./Habit.sol");
+
+module.exports = function(deployer) {
+    deployer.deploy(Habit, 1000);
```
Comment (dragontho) [inline backend/migrations/2_deploy_contracts.js:4]: Likewise, the 1000 here will be gone.

**Review by dragontho — COMMENTED** [review]
LGTM

**Review by ambhinav — COMMENTED** [review]
lgtm

**Review by gracelix — APPROVED** [review]
lgtm

**Pull request merged**